webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -288,25 +289,27 @@ const NavigationSideBar: React.FC = () => {
                   <p className={css.noWorkspaces}>No pinned workspaces</p>
                 ) : (
                   <ul className={css.pinnedWorkspaces} role="list">
-                    {workspaces.map((workspace) => (
-                      <WorkspaceActionDropdown
-                        key={workspace.id}
-                        trigger={['contextMenu']}
-                        workspace={workspace}>
-                        <li>
-                          <NavigationItem
-                            icon={<DynamicIcon name={workspace.name} size={24} />}
-                            label={workspace.name}
-                            labelRender={
-                              <Typography.Paragraph ellipsis={{ rows: 1, tooltip: true }}>
-                                {workspace.name}
-                              </Typography.Paragraph>
-                            }
-                            path={paths.workspaceDetails(workspace.id)}
-                          />
-                        </li>
-                      </WorkspaceActionDropdown>
-                    ))}
+                    {workspaces
+                      .sort((a, b) => numericSorter(a.id, b.id))
```

Comment:
idk which one is better but sort by name is more user friendly imo

Reply:
Agreed, I was surprised to see that we currently have it oldest -> newest. I will ping design

Reply:
After convo with design, changed this to sort by `pinned.createdAt` with newest pins at the bottom (luckily this was always in our DB and just needed to be sent by the API and can be sorted client-side)